students/chanjoo/westargram/users/views.py

```diff
@@ -63,12 +65,15 @@ def post(self, request):
             if not User.objects.filter(email=email).exists():
                 raise ValidationError('INVALID_EMAIL')
 
-            hashed_password  = User.objects.get(email=email).password.encode('utf-8')
+            user = User.objects.get(email=email)
 
-            if not bcrypt.checkpw(password.encode('utf-8'), hashed_password):
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                 raise ValidationError('INVALID_PASSWORD')
```

Comment:
email이든 비밀번호든 잘 못 되었을 때 에러 메시지를 통일하는 것이 좋습니다. 
email이 잘 못 되었는지, 비밀번호가 잘 못 되었는지 몰라야 보안상 더 좋습니다. 
```suggestion
            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                raise ValidationError('INVALID_USER')
```

Reply:
보안상 이유이군요 새롭게 알게 되었네요 감사합니다.